### PR TITLE
Tom add validation

### PR DESCRIPTION
**Pull request by tladendo**
fyi @indytechcook @vietnogi -- this adds validation to the 3 fields that require it.

### CODE REVIEW

**Comment by tladendo** [comment]
also @vietnogi -- I noticed that when there's a hash on the end of the path, which I'm using for `$anchorScroll`, the styling breaks. Do you know what's up with this?

**Comment by vietnogi** [comment]
fixed in https://github.com/appirio-tech/ap-work-client/pull/106

**Comment by vietnogi** [comment]
Each form should validate when clicking the next button (or on submit).

**Comment by vietnogi** [comment]
Perhaps we should use the constraint validation api, http://www.html5rocks.com/en/tutorials/forms/constraintvalidation/

File: src/client/app/submit-work/about-project/views/elevator-pitch.jade [hunk 1]
```diff
@@ -1,8 +1,12 @@
 h2 About your project
 
-form
+form(name="summaryForm")
```
Comment (vietnogi) [inline src/client/app/submit-work/about-project/views/elevator-pitch.jade]: is the name used for scrolling?  It should be by id, also Ive added this that may help: https://github.com/appirio-tech/ap-work-client/pull/106

Comment (tladendo) [inline src/client/app/submit-work/about-project/views/elevator-pitch.jade]: No, it's just for use with `ng-messages`

**Comment by indytechcook** [comment]
I agree on validation on next button at the latest.  Submit is to late.

**Comment by vietnogi** [comment]
Don't forget, a user can continue by hitting the "enter" key.

**Pull request merged**